apps/docusaurus/docs/reference/indexer.md

```diff
@@ -0,0 +1,389 @@
+| collection_id            | String | Unique identifier for the collection. Ex. "0xa815a9a09105973084bfc31530e7c8f002846787c2f0521e1e34dc144ad83b89"                         |
+| collection_name          | String | The formal name of the collection. Ex. "Mythic Dragons"                                                                                |
+| collection_uri           | String | URI linking to further information about the collection. Ex. "https://example.com/collections/9876543210"                              |
+| creator_address          | String | This is an Aptos account address that created the collection. Ex. "0x50bc83f01d48ab3b9c00048542332201ab9cbbea61bda5f48bf81dc506caa78a" |
+| current_collection       | Table  | Use the [Hasura explorer](#indexer-api-reference) to see these sub-fields.                                                             |
+| distinct_tokens          | bigint | The count of distinct tokens owned within this collection. Ex. 150                                                                     |
+| last_transaction_version | bigint | The version number of the last transaction involving this collection. Ex. 3000000002                                                   |
+| owner_address            | String | This is an Aptos account address that currently owns the token. Ex. "0x123abc456def7890abcdef1234567890abcdef1234"                     |
+| single_token_uri         | String | URI linking to information about a specific token within the collection. Ex. "https://example.com/tokens/9876543210"                   |
+| token_standard           | String | Aptos standard that the collection adheres to. Ex. "v1"                                                                                |
+
+## Fungible Assets
+
+### `fungible_asset_metadata`
+
+This tracks the metadata tied to each fungible asset (ex. decimals of precision). It includes v1 token data.
+
+| Field                             | Type   | Description                                                                                                                       |
+| --------------------------------- | ------ | --------------------------------------------------------------------------------------------------------------------------------- |
+| asset_type                        | String | The type of the asset, described by a Move resource. Ex. "0x1::aptos_coin::AptosCoin"                                             |
+| creator_address                   | String | This is an Aptos account address that created the asset. Ex. "0x50bc83f01d48ab3b9c00048542332201ab9cbbea61bda5f48bf81dc506caa78a" |
+| decimals                          | bigint | Number of decimal places for token value, typically for fungible tokens. Ex. 18                                                   |
```

Comment:
To get back the actual asset amount, divide the amount field by 1e{decimal}, e.g. amount / 1e8 to get APT from octa.